packages/panzer/disc-fe/src/Panzer_FaceToElement_impl.hpp

```diff
@@ -162,8 +162,14 @@ initialize(panzer::ConnManager<LocalOrdinal,GlobalOrdinal> & conn)
   Teuchos::RCP<GOMultiVector>  face2elem_mv = Teuchos::RCP<GOMultiVector>(new GOMultiVector(face_map, 8));
  // set a flag of -1-shift to identify unmodified data
   face2elem_mv->putScalar(-1-shift);
-  auto b1 = face2elem_mv->getDataNonConst(0), e1 = face2elem_mv->getDataNonConst(1), p1 = face2elem_mv->getDataNonConst(2), l1 = face2elem_mv->getDataNonConst(3);
-  auto b2 = face2elem_mv->getDataNonConst(4), e2 = face2elem_mv->getDataNonConst(5), p2 = face2elem_mv->getDataNonConst(6), l2 = face2elem_mv->getDataNonConst(7);
+  auto b1 = face2elem_mv->getDataNonConst(0);
+  auto e1 = face2elem_mv->getDataNonConst(1);
+  auto p1 = face2elem_mv->getDataNonConst(2);
+  auto l1 = face2elem_mv->getDataNonConst(3);
+  auto b2 = face2elem_mv->getDataNonConst(4);
+  auto e2 = face2elem_mv->getDataNonConst(5);
+  auto p2 = face2elem_mv->getDataNonConst(6);
+  auto l2 = face2elem_mv->getDataNonConst(7);
```

Comment:
Does CUDA not like multiple variable declarations per line?  Or is it because they're all type `auto`?

Reply:
It was auto+multiple declarations. It gave a warning about using auto in two different contexts. something I have never seen before.

Reply:
Good to know.  Haven't seen that before either.